lib/protocol/src/openctx-protocol.schema.ts

```diff
@@ -62,12 +76,38 @@ export interface Selector {
      */
     contentContains?: string
 }
-export interface ItemsParams {
+/**
+ * A mention contains presentation information relevant to a resource.
+ */
+export interface Mention {
+    /**
+     * A descriptive title.
+     */
+    title: string
+    /**
+     * A URI for the mention item.
+     */
+    uri: string
+    data?: {
+        [k: string]: unknown | undefined
```

Comment:
is there a better type here for the value? We want to make sure that whatever the type here is it is json serializable.

Reply:
This is generated code. Will figure out how we can update the JSON schema to generate a better type here later.